consensus/types/src/beacon_state.rs

```diff
@@ -911,6 +911,22 @@ impl<E: EthSpec> BeaconState<E> {
         }
     }
 
+    /// Returns the block root at the last slot of `epoch - 1`.
+    ///
+    /// This can be deleted after Glamsterdam and the removal of the v1 proposer duties endpoint.
+    pub fn legacy_proposer_shuffling_decision_root_at_epoch(
+        &self,
+        epoch: Epoch,
+        head_block_root: Hash256,
+    ) -> Result<Hash256, Error> {
+        let decision_slot = epoch.saturating_sub(1u64).end_slot(E::slots_per_epoch());
```

Comment:
I think this is fine, but just want to note that the previous logic uses `min_seed_lookahead`, but no live network modifies this config

https://github.com/sigp/lighthouse/blob/d67ae921125dd22e15d91aa324a27725ac03df41/consensus/types/src/chain_spec.rs#L894-L900